e2e/content-services/document-list/document-list-component.e2e.ts

```diff
@@ -342,7 +343,9 @@ describe('Document List Component', () => {
 
         afterAll(async () => {
             for (let i = 0; i <= folderCreated.length; i++) {
-                await uploadActions.deleteFileOrFolder(folderCreated[i].entry.id);
+                if (folderCreated[i]) {
```

Comment:
wouldn't it be more correct to have the `for` go until `i < folderCreated.length` instead of `<=` ?
if so, the `if` would no longer be needed

Reply:
Yes sure I didn't notice :)